frontend/integration-tests/views/crud.view.ts

```diff
@@ -7,7 +7,8 @@ import { appHost, testName, waitForNone } from '../protractor.conf';
 import { waitForCount } from '@console/shared/src/test-utils/utils';
 
 export const createYAMLButton = $('#yaml-create');
-export const createItemButton = $('#item-create');
```

Comment:
are you sure this change is correct for all tests that use this const ?

Reply:
Old selector doesn't exist any more, so it is OK to change this